crates/bevy_ui/src/lib.rs

```diff
@@ -20,15 +20,21 @@ pub mod prelude {
 }
 
 use bevy_app::prelude::*;
-use bevy_ecs::{IntoSystem, ParallelSystemDescriptorCoercion, SystemStage};
+use bevy_ecs::{
+    IntoSystem, ParallelSystemDescriptorCoercion, StageLabel, SystemLabel, SystemStage,
+};
 use bevy_render::render_graph::RenderGraph;
 use update::ui_z_system;
 
 #[derive(Default)]
 pub struct UiPlugin;
-
-pub mod stage {
-    pub const UI: &str = "ui";
+#[derive(Debug, Hash, PartialEq, Eq, Clone, StageLabel)]
```

Comment:
In general enum type names should be singular (and i personally think that applies to stage labels);